UofS-Pulse-Binfo/rawphenotypes
# Make queries more specific; reduce LIMIT 1

While testing the Tripal 3 upgrade, I came across some bugs caused by the use of "LIMIT 1" when pulling out cvterms (#33). Since no ordering is applies, using limit 1 can produce inconsistent results if there are two terms matching the filter criteria. As such, I made these queries more specific and took advantage of database constraints to ensure a unique record is pulled out.

## Testing
- Unfortunately these queries touch much of the non-admin UI...
- Check that the download/upload pages function as expected
- Inspect the queries visually
